tools/sleigh-lift/src/main.cpp

```diff
@@ -1,5 +1,10 @@
-#include <sleigh/loadimage.hh>
-#include <sleigh/sleigh.hh>
+#pragma GCC diagnostic push
```

Comment:
Hmm, do you want the header to include every other header under `sleigh/`? I agree though, this would be useful. It's annoying to get this to work across GCC/Clang/MSVC so it'd be good to have it in the lib so that it can be reused.

Reply:
GCC and Clang will both work with `#pragma GCC diagnostic push` and `pop`, so it's more about MSVC.

Reply:
Sure. What about this?
> Hmm, do you want the header to include every other header under sleigh/?

Reply:
Sure.